commit 5e9668ccaf3d34567b313b13327ff9b31e4e5ba6
Author: Mark Wielaard <mjw@redhat.com>
Date:   Fri Jul 4 13:26:12 2014 +0200

    backends: aarch64_return_value_location should handle DW_ATE_boolean.
    
    Found with run-native-test.sh om debian arm64.
    http://bugs.debian.org/753552
    
    Signed-off-by: Mark Wielaard <mjw@redhat.com>

diff --git a/backends/aarch64_retval.c b/backends/aarch64_retval.c
index 0ed7d56..68de307 100644
--- a/backends/aarch64_retval.c
+++ b/backends/aarch64_retval.c
@@ -357,6 +357,7 @@ aarch64_return_value_location (Dwarf_Die *functypedie, const Dwarf_Op **locp)
 		 size of the argument is less than or equal to 8 bytes
 		 [...] the argument is copied to the least significant
 		 bits in x[NGRN].  */
+	    case DW_ATE_boolean:
 	    case DW_ATE_signed:
 	    case DW_ATE_unsigned:
 	    case DW_ATE_unsigned_char:
